src/config.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto2";
+
+package tutorial;
+
+message Person {
```

Comment:
Does this remain from example run? Please remove.

Reply:
This was for proto PR. I think I switched branches a little late that config.proto is still in this PR. I will remove